deploy/tm-load-test/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:latest
+
+RUN apt-get update -y -q && apt-get upgrade -yq
+# common (DEBIAN_FRONTEND is a fix for tzdata)
+RUN DEBIAN_FRONTEND="noninteractive" apt-get install --no-install-recommends -yq software-properties-common \
+                                                curl \
+                                                build-essential \
+                                                ca-certificates \
+                                                tar \
+                                                git
+
+# ------------------ onomy chain ------------------
+
+ARG ONOMY_DIR=/go/src/github.com/onomyprotocol/onomy/
+# clone the repo
+COPY . $ONOMY_DIR
+
+# install golang
+RUN curl https://dl.google.com/go/go1.16.4.linux-amd64.tar.gz --output go.tar.gz
```

Comment:
you can use go based image instead of ubuntu, it will decrease the time for the test

Reply:
yeah sure.